ansible/ss_cluster/roles/slave/tasks/serverconf_slave_db.yml

```diff
@@ -21,7 +21,9 @@
       dest: "{{ postgresql_data_dir }}/recovery.conf"
       owner: postgres
       group: postgres
-  when: ansible_distribution_release == "bionic"
+  when: >
```

Comment:
Hi, thank you for creating this PR!

We discussed this in the team and thought that it would be better to compare postgresql version instead of Linux distribution version. This would be a bit more clear to the maintainer of this playbook, and more robust e.g. in case we e.g. run the playbook in new distribution versions in future.

Alternative implementation is at https://github.com/nordic-institute/X-Road/pull/969/files - have a look at it, you can backport the changes to your PR, or we could merge 969 directly. 

I tested PR 969 on Ubuntu 18/20 and Centos 7/8. On a related not, running on centos brought into light some bugs, they were fixed in https://github.com/nordic-institute/X-Road/pull/968

Reply:
@gudvardur I will proceed to merge the alternative implementation https://github.com/nordic-institute/X-Road/pull/969/files now